snapshots/smoking/2024-05-30/cigarette_sales.zip.dvc

```diff
@@ -0,0 +1,28 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: International Smoking Statistics
+    date_published: "2016-12-19"
```

Comment:
Oh wow! I didn't know that this dataset has not been updated since 2016! In such cases, it's good to consider whether putting much effort into the dataset is worth it. Have you checked if there is an alternative, more recent dataset on this topic elsewhere?

Reply:
Hi @pabloarosado — there's a lot of hidden context behind this update, all explained [here](https://github.com/owid/owid-issues/issues/1490). In a nutshell, @spoonerf and I specifically selected this dataset because we really need to fix it _and_ ti's good practice for @antea04. So no worries here!

Reply:
Like Ed said (and mentioned via Slack) - as an exercise and also since this data set is referenced in a few different places around the site it was useful to fix it - but I do agree that more recent data would also be interesting!